typedoc.json

```diff
@@ -0,0 +1,33 @@
+{
+	"$schema": "https://typedoc.org/schema.json",
+	"categoryOrder": [
+		"Node Types",
+		"Nodes - Type Guards",
+		"Nodes - Flag Utilities",
+		"Nodes - Other Utilities",
+		"Modifier Utilities",
+		"Scope Utilities",
+		"Syntax Utilities",
+		"Compiler Options",
+		"*",
+		"Other"
+	],
+	"cleanOutputDir": true,
+	"defaultCategory": "Other",
+	"entryPointStrategy": "resolve",
+	"entryPoints": ["./src/index.ts"],
+	"exclude": ["./**/test/", "./**/*.test.*"],
+	"excludeExternals": true,
+	"excludeInternal": true,
+	"excludeNotDocumented": false,
+	"excludePrivate": true,
```

Comment:
[Question] Is there a way to hide the _Modules_ section from the nav bar? It's only ever going to be one in this package.

![Screenshot of the Modules nav section with the one member "ts-api-utils - v0.0.22"](https://user-images.githubusercontent.com/3335181/219901554-859bf3a9-e242-41eb-bbb3-ee617db0b2e6.png)

[Request] If there is, can we do it here? If not, we should file a followup issue to track requesting an option for it.

Reply:
I don't think there is. At least now without a plugin.